atlas/AtlasMl/atlasml/clients/artemis_client.py

```
@@ -0,0 +1,212 @@
+        """Check if Artemis API is available.
+
+        Returns:
+            True if API is healthy, False otherwise
+        """
+        try:
+            async with httpx.Client(timeout=10.0) as client:
+                response = client.get(
+                    f"{self.base_url}/api/health",
+                    headers={"Accept": "application/json"}
+                )
+                return response.status_code == 200
```

Ruff (0.12.2)

Magic value used in comparison, consider replacing `200` with a constant variable

(PLR2004)